app-editors/vim/vim-9.1.0.recipe

```diff
@@ -0,0 +1,148 @@
+SUMMARY="A highly configurable text editor"
+DESCRIPTION="Vim is a highly configurable text editor built to enable \
+efficient text editing. It is an improved version of the vi editor \
+distributed with most UNIX systems.
+
+Vim is often called a \"programmer's editor,\" and so useful for programming \
+that many consider it an entire IDE. It's not just for programmers, though. \
+Vim is perfect for all kinds of text editing, from composing email to editing \
+configuration files.
+Despite this, Vim can be configured to work in a very simple (Notepad-like) \
+way, called evim or Easy Vim."
+HOMEPAGE="https://www.vim.org/"
+COPYRIGHT="1991-2023 Bram Moleenar et al."
+LICENSE="Vim"
+REVISION="1"
+SOURCE_URI="https://github.com/vim/vim/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="ddb435f6e386c53799a3025bdc5a3533beac735a0ee596cb27ada97366a1c725"
+SOURCE_FILENAME="vim-$portVersion.tar.gz"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
```

Comment:
Any specific reason why to remove the variable $pythonVersion here?
Also, remove older recipe so it's easier to see what's changed.